server/app/routes/files.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const expressFileUpload = require('express-fileupload');
+
+const { upload } = require('../controllers/files');
+const { protect } = require('../middleware/auth');
+
+const router = express.Router();
+
+router.use(expressFileUpload());
```

Comment:
The expressFileUpload middleware needs to go inside app.js and BEFORE the endpoint routes, otherwise the routers won't have access to it.